crates/router/src/services/api.rs

```diff
@@ -351,10 +364,48 @@ where
             match connector_request {
                 Some(request) => {
                     logger::debug!(connector_request=?request);
+
+                    let masked_request_body = match &request.body {
+                        Some(request) => match request {
+                            RequestContent::Json(i)
+                            | RequestContent::FormUrlEncoded(i)
+                            | RequestContent::Xml(i) => i
+                                .masked_serialize()
+                                .unwrap_or(json!({ "error": "failed to mask serialize"})),
+                            RequestContent::FormData(_) => json!({"request_type": "FORM_DATA"}),
+                        },
+                        None => json!({"error": "EMPTY_REQUEST_BODY"}),
```

Comment:
in case of Get calls to the connector, body will be always empty. So instead of passing random json let's pass Null